Bottom: 0f89b6cd75fb2a8999ef9228da1db57e75bb8579
Top:    07b417a182c3180fdbe391fe407ffdcb4c5c273c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 15:06:40 +0100

Refresh of hid-gtk-remove-dead-code-due-t

---

diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index 6cd8069..2a205fa 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -52,7 +52,7 @@
 RCSID ("$Id$");
 
 
-static gboolean
+static void
 pinout_zoom_fit (GhidPinoutPreview * pinout, gint zoom)
 {
   pinout->zoom = zoom;
@@ -69,11 +69,6 @@ pinout_zoom_fit (GhidPinoutPreview * pinout, gint zoom)
   pinout->h_pixels = (gint) (pinout->scale *
 			     (pinout->element.BoundingBox.Y2 -
 			      pinout->element.BoundingBox.Y1));
-
-  if (pinout->w_pixels > 3 * Output.Width / 4 ||
-      pinout->h_pixels > 3 * Output.Height / 4)
-    return FALSE;
-  return TRUE;
 }
 
 
@@ -128,8 +123,7 @@ pinout_set_data (GhidPinoutPreview * pinout, ElementType * element)
 		       Settings.PinoutOffsetY -
 		       pinout->element.BoundingBox.Y1);
 
-  if (!pinout_zoom_fit (pinout, 2))
-    pinout_zoom_fit (pinout, 3);
+  pinout_zoom_fit (pinout, 3);
 
   ELEMENTLINE_LOOP (&pinout->element);
   {
